packages/flutter_tools/lib/src/commands/update_packages.dart

```diff
@@ -26,7 +26,8 @@ const Map<String, String> _kManuallyPinnedDependencies = <String, String>{
   'test_api': '0.2.5',     //  |
   'test_core': '0.2.5',    //  |
   'build_runner': '1.6.1', // TODO(jonahwilliams): allow newer versions
-  'build_modules': '2.3.0'
+  'build_modules': '2.3.0',
+  'vm_service_client': '0.2.6+2', // Final version before being marked deprecated.
```

Comment:
Worth adding a comment that this should be replaced w/ `vm_service`

Reply:
Done.  I've filed https://github.com/flutter/flutter/issues/37499 and added a TODO.